cmd/crc/cmd/start.go

```diff
@@ -262,18 +262,39 @@ If you find an issue, please report it at https://github.com/openshift/okd
 `
 )
 
+func startTemplateForMicroshift() string {
+	tmpl := `Started the OpenShift edge cluster.
```

Comment:
The product name is apparently "Red Hat build of MicroShift" https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.12 I'm not sure "OpenShift edge cluster" is an accurate description. Just `Started the MicroShift cluster`?

Reply:
The other name I've seen used is "Red Hat Device Edge" https://www.redhat.com/en/technologies/device-edge

Reply:
@cfergeau this is something I need clarification from @gbraad because yesterday also we were discussing it, should it be `Openshift edge` or `microshift. I am also bit align to use `microshift`.

Reply:
We will use `microshift` as the preset name to choose it from our tooling, but what the UI does can be different. Perhaps we might add an alias to this later, but microshift is for now the choice.

Reply:
The question was about what to show in the UI, `microshift` as a preset name is not under discussion.
I'm not seeing any official use of "OpenShift for Edge" (and should it be "Red Hat OpenShift for Edge" then?)
I'm fine with `Started the MicroShift cluster` for now.